src/metadata/cities_to_county.csv

```diff
@@ -0,0 +1,456 @@
+﻿County,City
```

Comment:
This is a nit, but I think this file would be easier to follow and diff any changes if the column order was `City,County` and/or if it was sorted by County first, then City.